### ISSUE TITLE
Loading vector map failed

### ISSUE THREAD

**Issue by cyberphysicalrobotics** [Autoware-AI/common#11]
### Required information:

- Operating system and version:
  - Ubuntu 18.04

- Autoware installation type:
  - Docker with mounted source code directory
- Autoware version or commit hash
  - Autoware common package commit hash is 1fa376783ff584528510587bf80257dafd6d6fb3
- ROS distribution and version:
  - melodic
- ROS installation type:
  - Docker
- Package or library, if applicable:
  - 

### Description of the bug
Loading point maps succeeded.
Loading vector map failed with the following error.
 
```
['roslaunch', '/home/autoware/shared_dir/autoware-data/BorregasAve/my_launch/my_map.launch']
pid=21670
sched policy=OTHER prio=0
load /home/autoware//shared_dir/autoware-data/BorregasAve/data/map/pointcloud_map/BorregasAve.pcd
/home/autoware/Autoware/install/map_file/lib/map_file/vector_map_loader: symbol lookup error: /home/autoware/Autoware/install/map_file/lib/map_file/vector_map_loader: undefined symbol: _ZN10vector_map9VectorMap9subscribeERN3ros10NodeHandleEy
[vector_map_loader-4] process has died [pid 21702, exit code 127, cmd /home/autoware/Autoware/install/map_file/lib/map_file/vector_map_loader /home/autoware//shared_dir/autoware-data/BorregasAve/data/map/vector_map/lane.csv /home/autoware//shared_dir/autoware-data/BorregasAve/data/map/vector_map/dtlane.csv /home/autoware//shared_dir/autoware-data/BorregasAve/data/map/vector_map/point.csv /home/autoware//shared_dir/autoware-data/BorregasAve/data/map/vector_map/line.csv /home/autoware//shared_dir/autoware-data/BorregasAve/data/map/vector_map/pole.csv /home/autoware//shared_dir/autoware-data/BorregasAve/data/map/vector_map/signaldata.csv /home/autoware//shared_dir/autoware-data/BorregasAve/data/map/vector_map/stopline.csv /home/autoware//shared_dir/autoware-data/BorregasAve/data/map/vector_map/vector.csv /home/autoware//shared_dir/autoware-data/BorregasAve/data/map/vector_map/whiteline.csv /home/autoware//shared_dir/autoware-data/BorregasAve/data/map/vector_map/node.csv __name:=vector_map_loader __log:=/home/autoware/.ros/log/3fd75d90-d746-11ea-b6c2-54bf647d3a59/vector_map_loader-4.log].
log file: /home/autoware/.ros/log/3fd75d90-d746-11ea-b6c2-54bf647d3a59/vector_map_loader-4*.log

(runtime_manager_dialog.py:21408): Gtk-WARNING **: 18:06:32.942: Negative content width -2 (allocation 32, extents 17x17) while allocating gadget (node button, owner GtkToggleButton)
```

### Steps to reproduce the bug
1.Download vector, point map.
```
$ cd ~/shared_dir
$ git clone https://github.com/lgsvl/autoware-data.git
```
2.Run docker container
```
# Change directory to docker/generic
$ ./run.sh -t local # in my case, I used local tag. Instead you may use 1.14.0 tag.
```
3.Apply commit to make bug.
```
# Change directory to autoware common one.
$ git reset --hard 1fa376783ff584528510587bf80257dafd6d6fb3
# In df8f9fa commit, loading vector map succeeded, In 1fa3767 commit, it failed.
```
4.Start Runtime Manager
```
$ roslaunch runtime_manager runtime_manager.launch
```
5.Select map launch file.
```
/home/autoware/shared_dir/autoware-data/BorregasAve/my_launch/my_map.launch
```
6.Click Map in Quick Start tab and start Rviz

### Expected behavior
Vector map should be loaded like point map is loaded.

### Actual behavior
Vector map is not loaded with the error.

### Screenshots
The following shows that loading vector map failed.
![image](https://user-images.githubusercontent.com/14055939/89449184-9609e900-d70d-11ea-904d-7451e32ab60b.png)

**Comment by JWhitleyWork** [comment]
@cyberphysicalrobotics I could not reproduce your described bug. Here are the exact steps I took:

```
$ cd ~
$ git clone https://github.com/Autoware-AI/docker.git
$ cd shared_data/
$ git clone https://github.com/lgsvl/autoware-data.git
$ cd ~/docker/generic
$ ./run.sh -t 1.14.0 -c off
autoware$ source Autoware/install.bash
autoware$ roslaunch runtime_manager runtime_manager.launch
```

1. On the Quick Start tab, click Ref next to Map button and select `/home/autoware/shared_dir/autoware-data/BorregasAve/my_launch/my_map.launch`
2. Click the Map button
3. Click the RViz button

When RViz opens, I see the vector map but not the point cloud map. If I do each of the steps manually in the `Map` tab that are combined in the `my_map.launch` file, I see both the vector map and the point cloud map. However, I did not see an error in the terminal window relating to `vector_map_loader` dying.

To test with the commit you suggest, I did the following:

```
autoware$ cd ~/Autoware/src/autoware/common
autoware$ git reset --hard 1fa376783ff584528510587bf80257dafd6d6fb3
autoware$ cd ../../
autoware$ colcon build
```

However, when I tried to build the repository I got an error in `core_planning/lane_planner`. It appears that Lanelet2 has changed their API since 1.14.0 and the build does not succeed. I'm not sure how you were able to build the stack without this failure.

Just to make sure you are aware: changing the commit in the repository without re-building does not change the code that is run with the `roslaunch` command.
